test/ctia/test_helpers/core.clj

```diff
@@ -511,3 +512,17 @@
       (f)
       (reset! uuid-counter
               uuid-counter-start))))
+
+(defn naive-longest-common-suffix
+  "O(|strs| * shortest string)"
+  [strs]
+  (when-not (seq strs)
+    (throw (ex-info "non-empty strs needed" {})))
```

Comment:
what about assert?
```suggestion
  (assert (not-empty strs) "non-empty strs needed")
```

Reply:
I don't want this check being turned off (assertions can be disabled) so I used an explicit throw.